tox.ini

```
@@ -27,6 +27,8 @@ commands =
     --cov-report=xml \
     --cov=fiasco \
     {toxinidir}/docs \
+    --durations=10 \
```

```suggestion
    --durations=20 \
```
...in case you'd like more extensive durations reporting.